robolectric/src/test/java/org/robolectric/shadows/ShadowBitmapFactoryTest.java

```diff
@@ -142,6 +142,18 @@ public void decodeBytes_shouldSetDescriptionAndCreatedFrom() throws Exception {
     assertEquals(100, bitmap.getHeight());
   }
 
+  @Test
+  public void decodeBytesWithOptions_shouldSetDescriptionAndCreatedFrom() throws Exception {
```

Comment:
What about changing name to `decodeBytes_shouldSetDescriptionAndCreatedFromWithOptions` and squashing all commits to one?

Reply:
@utzcoz I updated the name, I am not sure how to configure this PR to be squashed into a single commit. I normally use gitlab.